topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/kafka/KafkaTestConfiguration.java

```diff
@@ -83,7 +83,8 @@ static PropertiesConfigurer propertiesConfigurer(final ResourceLoader resourceLo
     }
 
     @Bean
-    @SuppressWarnings("java:S112") // Suppress specific exception smell
+    @SuppressWarnings("java:S112")
+        // Suppress specific exception smell
```

Comment:
```suggestion
    // Suppress specific exception smell
    @SuppressWarnings("java:S112")
```
I think the comment should describe the smell that's being suppressed